src/dmd/statementsem.d

```diff
@@ -3795,7 +3795,7 @@ else
 
         if (!global.params.useExceptions)
         {
-            ts.error("Cannot use throw statements with -betterC");
+            ts.error("Cannot use `throw` statements with -betterC");
```

Comment:
OT: Now that `betterC` is split up and the parameters might be exposed separately, this error message isn't ideal.

Reply:
Noted here: https://github.com/dlang/dmd/projects/3#card-6872539